python/lsst/ip/isr/isrFunctions.py

```diff
@@ -753,14 +753,19 @@ def applyGains(exposure, normalizeGains=False):
     normalizeGains : `Bool`, optional
         If True, then amplifiers are scaled to force the median of
         each amplifier to equal the median of those medians.
+    ptcGains : `dict`[`str`], optional
+        Dictionary keyed by amp number containing the PTC gains.
```

Comment:
From the looks of the rest of the code, I think this is actually the amp name, not the amp number (and that's better, much less ambiguity/potential for iteration mistakes that way), so I think just change the doc here.